dbt_coves/utils/airbyte_api.py

```diff
@@ -0,0 +1,114 @@
+from typing import Dict
+from requests.exceptions import RequestException
+import requests, json
+
+
+class AirbyteApiCallerException(Exception):
+    pass
+
+
+class AirbyteApiCaller:
+    def api_call(self, endpoint: str, body: Dict[str, str] = None):
+
+        """
+        Generic `api caller` for contacting Airbyte
+        """
+        try:
+            response = requests.post(endpoint, json=body)
+            if response.status_code == 204:
```

Comment:
Please keep in mind what I always tell you: functions should never change their return `type`. It could generate unexpected issues.

Reply:
031de77[](url)